demo/src/screens/AboutScreen/AboutScreen.js

```diff
@@ -1,12 +1,41 @@
 import React from 'react';
+import {GoQuestion} from 'react-icons/go';
+import './AboutScreen.css';
+
+const /** JSX */ DESCRIPTION = <p>{'This page is a sample ecommerce ' +
+'application with a complete implementation of '}
```

Comment:
This is probably me not understanding JSX but why <p>{'Text'} instead of <p>Text
https://reactjs.org/docs/introducing-jsx.html#embedding-expressions-in-jsx

for multiline html it seems that wrapping with () should be sufficient 
https://reactjs.org/docs/introducing-jsx.html#jsx-represents-objects

Reply:
Some places consider it a best practice to use {'text'} all the time since it can handle special characters and whitespace a bit better. I don't think that's a  very common practice (I didn't see it in the airbnb style guide for example), and it really clutters up this screen without providing much benefit, so I took it out for the most part. I left it inside of the <a> tags due to whitespace issues.